src/interfaces/history-item-interface.ts

```diff
@@ -53,6 +53,7 @@ export interface HistoryItemIF {
   fromLegalType?: CorpTypeCd
 
   // dissolutions only
+  dissolutionDate?: IsoDatePacific
```

Comment:
This is one of our own types:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/date-interfaces.ts